iConverter/Models/Base/iConverterError.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  iConverterError.swift
+//  iConverter
+//
+//  Created by Aram Semerjyan on 21.04.22.
+//
+
+import Foundation
+
+enum iConverterError: Error {
```

Comment:
Seems that `iConverterError` acts as namespace so no need to conform to `Error` ?